site/data/roadmap.yaml

```diff
@@ -43,28 +37,52 @@
   url: /blog/2023-01-19-authentication-help/
   year: n/a
 
+- item: Browser Recorder
+  status: 🎉 Finished
+  url: https://summerofcode.withgoogle.com/programs/2023/projects/lx6XL1Rd
+  sponsor: Google Summer of Code
+  sponsorlink: https://summerofcode.withgoogle.com/
+  year: 2023
+  archive: true
+- item: Import Postman API Definitions into ZAP
+  status: 🎉 Finished
+  url: https://summerofcode.withgoogle.com/programs/2023/projects/OlBxaE5X
+  sponsor: Google Summer of Code
+  sponsorlink: https://summerofcode.withgoogle.com/
+  year: 2023
+  archive: true
+- item: "OpenSSF Best Practices: Silver"
+  status: 🎉 Finished
```

Comment:
Not quite - we're at 98% 😁

Reply:
Okay, maybe this year we'll get to 100% :) Tweaked.